environment.yml

```diff
@@ -3,26 +3,24 @@ channels:
   - conda-forge
   - desilinguist
 dependencies:
-- python==3.6.3
-- ipython=6.3.1
-- jupyter=1.0.0
-- joblib=0.11
-- matplotlib=2.1.2
-- nose=1.3.7
-- notebook=5.4.1
-- numpy=1.13.3
-- pandas=0.22.0
-- scikit-learn=0.19.1
-- scipy=1.0.0
-- seaborn=0.9.0
-- skll=1.5.2
-- statsmodels=0.9.0
-- parameterized
-- zeromq
-- sphinx
-- sphinx_rtd_theme
-- coverage
-- setuptools
-- openpyxl
-- xlrd
-- xlwt
+  - python=3.6.8
+  - ipython=6.5.0
+  - jupyter=1.0.0
+  - joblib=0.11
+  - matplotlib=2.1.2
+  - nose=1.3.7
+  - notebook=5.7.2
+  - numpy=1.14.0
```

Comment:
Do we need to pin numpy? Since this is only used for readthedocs, using the latest version of numpy should be fine right? And same for `pandas` etc.?